plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -112,21 +113,38 @@ export class PersonState {
 
     async update(): Promise<Person> {
         if (!this.processPerson) {
-            // We don't need to handle any properties for `processPerson=false` events, so we can
-            // short circuit by just finding or creating a person and returning early.
-            //
-            // In the future, we won't even get or create a real Person for these events, and so
-            // the `processPerson` boolean can be removed from this class altogether, as this class
-            // shouldn't even need to be invoked.
-            const [person, _] = await promiseRetry(() => this.createOrGetPerson(), 'get_person_personless')
+            if (this.lazyPersonCreation) {
+                const person = await this.db.fetchPerson(this.teamId, this.distinctId, { useReadReplica: true })
```

Comment:
I considered putting an LRU in front of this, but if we're going to remove ordering from this lane then I wonder how large this will have to be to have a decent hit rate? We could also cache in Redis, but we're already hitting the PG read replica...

The biggest win, I think, will come from a larger refactoring so that we can do all of the lookups for a batch at once.

Curious what folks think, though.